css/style.css

```diff
@@ -49,23 +67,19 @@ body {
 
 .sidenav {
    height: 100%; /* 100% Full-height */
-   width: 0; /* 0 width - change this with JavaScript */
+   width: 200px;
```

Comment:
Then thing was that while the sidenav was growing, as you can see in the following picture menu items had a weird behaviour:

![captura de pantalla 2016-06-12 10 56 54](https://cloud.githubusercontent.com/assets/1859128/15990518/ac231940-308c-11e6-9ec0-534f332399df.png)

Some items had 2 lines while growing, and also part of the text was sliced/cutted...

On the other hand, shrink the main div did that our profiles were collapsed, so now, sidenav goes over the main div, it also *auto-hides* when you click on one of the sidenav items.